hack/build-cross.sh

```diff
@@ -20,6 +25,8 @@ OS_BUILD_PLATFORMS=("${platforms[@]}")
 host_platform=$(os::build::host_platform)
 if [[ $host_platform == "linux/ppc64le" ]]; then
   OS_GOFLAGS_LINUX_PPC64LE="-tags=gssapi" os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+elif [[ $host_platform == "linux/arm64" ]]; then
```

Comment:
We can't assume Bash 3.4+ so we don't have associative arrays, so the generic approach in [`common.sh`](https://github.com/openshift/origin/blob/master/hack/common.sh#L321-L324) uses `eval`, but I still prefer it to these types of `if/elif` blocks.